### DISCUSSION TITLE
batch script modifications

### DISCUSSION THREAD

**Discussion by kbrrhari** [hafs-community/HAFS#196] [Q&A]
Dear All,

I am trying to set up HAFS for the Indian Ocean region. I have installed HAFS successfully on the Cray XC40 machine.

When I am running the script cronjob_hafsv1_hfsa.sh, it is creating some temporary qsub scripts in a directory.  
I want to change "#PBS -l select=1:mpiprocs=36:ncpus=36" to "#PBS -l nodes=2:ppn=36" in the qsub script which are auto-generated. In which file do I need to edit? 

Thanks in advance
Hari

**Comment by BijuThomas-NOAA** [comment]
Hari: Look sites/${machine_id}.ent file and look for relevant entities

**Reply by kbrrhari** [reply]
Thank you for your reply. Our HPC support has resolved this issue.
